core/src/main/java/org/opensearch/sql/expression/DSL.java

```diff
@@ -719,6 +719,10 @@ public FunctionExpression match_bool_prefix(Expression... args) {
     return compile(BuiltinFunctionName.MATCH_BOOL_PREFIX, args);
   }
 
+  public FunctionExpression wildcard_query(Expression... args) {
```

Comment:
After https://github.com/opensearch-project/sql/pull/1085 all `DSL` methods should be static. Please, rebase and update.

Reply:
Thanks. Addressed in fede123e5f94b27ba02f6baa5c362d744e63759f